res/css/structures/_RightPanel.scss

```
@@ -103,44 +103,59 @@ limitations under the License.
     mask-position: center;
 }
 
-$dot-size: 8px;
+$dot-size: 7px;
 $pulse-color: $alert;
 
 .mx_RightPanel_pinnedMessagesButton {
     &::before {
         mask-image: url('$(res)/img/element-icons/room/pin.svg');
         mask-position: center;
     }
-
-    .mx_RightPanel_pinnedMessagesButton_unreadIndicator {
+}
+.mx_RightPanel_headerButton_unreadIndicator_bg {
+    content: "";
+    position: absolute;
+    right: 0;
+    top: 0;
+    margin: 4px;
+    width: $dot-size;
+    height: $dot-size;
+    border-radius: 50%;
+    transform: scale(1.6);
+    transform-origin: center center;
+    background: rgba($background, 1);
+}
+.mx_RightPanel_headerButton_unreadIndicator {
```

newline to make blocks easier to read
```suggestion

.mx_RightPanel_headerButton_unreadIndicator {
```